lr-cli/cortx_setup/commands/node/finalize.py

```diff
@@ -215,6 +215,29 @@ def _create_field_users(self, node_id):
                     targets=node_id
                 )
 
+            else:
+                self.logger.debug(f"Ensuring {user} user permissions on system")
+
+                StateFunExecuter.execute(
+                    'file.append',
+                    fun_kwargs=dict(
+                        name=f'{home_dir}/.bashrc',
+                        text='alias nodecli="sudo nodecli"'
+                    ),
+                    targets=node_id
+                )
+
+                StateFunExecuter.execute(
+                    'file.append',
+                    fun_kwargs=dict(
+                        name=f'{home_dir}/.profile',
+                        text=('sudo /usr/bin/nodecli \n'
+                              f'source {home_dir}/.bashrc')
```

Comment:
Do we really need to source bashrc?

Reply:
yes, sourcing rc file on profile login ensures our alias changes are in place.